api/src/test/java/org/apache/iceberg/TestHelpers.java

```diff
@@ -667,4 +671,40 @@ public List<Long> splitOffsets() {
       return null;
     }
   }
+
+  public static class PartitionSpecBuilder {
```

Comment:
Maybe we should have it be "ExpectedSpecBuilder" to avoid confusion with actual PartitionSpecs?

Reply:
OK.